docs/wcs/index.rst

```diff
@@ -113,6 +113,7 @@ Examples creating a WCS programmatically
    Example of Imaging WCS <example_create_imaging.rst>
    Example of Cube WCS <example_cube_wcs.rst>
    Loading From a FITS File <loading_from_fits.rst>
+   Creating a Planetary WCS <creating_planetary_wcs.rst>
```

Comment:
I think I would put it one line up, and maybe call it "Example of a Planetary WCS" since it is a similar example.